src/CommunityToolkit.Aspire.Hosting.MassTransit.RabbitMQ/MassTransitRabbitMQHostingExtensions.cs

```diff
@@ -0,0 +1,80 @@
+﻿using Aspire.Hosting;
+using Aspire.Hosting.ApplicationModel;
+using Microsoft.Extensions.Configuration;
+
+namespace CommunityToolkit.Aspire.Hosting.MassTransit.RabbitMQ;
+    
+/// <summary>
+/// MassTransitHostingExtensions provides extension methods for configuring MassTransit in an Aspire-based application.
+/// </summary>
+public static class MassTransitHostingExtensions
+{
+    /// <summary>
+    /// AddMassTransit configures a RabbitMQ-backed MassTransit integration in an Aspire-based application.
+    /// </summary>
+    /// <param name="builder">The application builder.</param>
+    /// <param name="name">A unique name for the RabbitMQ instance.</param>
+    /// <param name="configure">An optional configuration action to override default settings.</param>
+    public static IResourceBuilder<RabbitMQServerResource> AddMassTransit(this IDistributedApplicationBuilder builder,
+        string name, Action<MassTransitOptions>? configure = null)
```

Comment:
```suggestion
    public static IResourceBuilder<RabbitMQServerResource> AddMassTransit(this IDistributedApplicationBuilder builder, [ResourceName] string name, Action<MassTransitOptions>? configure = null)
```

Ensure that we have the `ResourceName` attribute on the name parameter for a resource.